tools/topology/topology2/include/components/copier.conf

```diff
@@ -19,6 +19,8 @@ Class.Widget."copier" {
 	#
 	DefineAttribute."index" {}
 
+	# dai_index is the BE DAI index for DAI copiers, FE DAI index (same as the PCM ID) for
```

Comment:
them why not use the same value to FE DAI index and PCM ID with a macro or something that enforces this rule?

Reply:
yes, I think I should do that for my own sanity and others' as well